.meta-updater/src/index.ts

```diff
@@ -92,7 +92,7 @@ async function updateTSConfig (
         manifest.name === '@pnpm/package-requester'
       )
     ) {
-      // This is to avoid a circular graph (which TypeScript references do not support.
+      // This is to avoid a circular graph (which TypeScript references to do not support.
```

Comment:
I don't think this is correct

```suggestion
      // This is to avoid a circular graph (which TypeScript references do not support.
```